crates/ide-db/src/search.rs

```diff
@@ -447,6 +449,11 @@ impl<'a> FindUsages<'a> {
         self
     }
 
+    pub fn with_rename(mut self, rename: Option<&'a Rename>) -> Self {
```

Comment:
This should be a temporary thing. Ideally this would trivially fix itself once we handle renamed `use`s as proper `Definition`s

Reply:
Oh, that's right. This should be handled as TypeAlias, right? I'll add a comment and modify the commit and PR message